doc/topics/reactor/index.rst

```diff
@@ -90,71 +87,278 @@ data structure and compiler used for the state system is used for the reactor
 system. The only difference is that the data is matched up to the salt command
 API and the runner system.  In this example, a command is published to the
 ``mysql1`` minion with a function of :py:func:`state.apply
-<salt.modules.state.apply_>`. Similarly, a runner can be called:
+<salt.modules.state.apply_>`, which performs a :ref:`highstate
+<running-highstate>`. Similarly, a runner can be called:
 
 .. code-block:: jinja
 
     {% if data['data']['custom_var'] == 'runit' %}
     call_runit_orch:
       runner.state.orchestrate:
-        - mods: _orch.runit
+        - args:
+          - mods: orchestrate.runit
     {% endif %}
 
 This example will execute the state.orchestrate runner and intiate an execution
-of the runit orchestrator located at ``/srv/salt/_orch/runit.sls``. Using
-``_orch/`` is any arbitrary path but it is recommended to avoid using "orchestrate"
-as this is most likely to cause confusion.
+of the ``runit`` orchestrator located at ``/srv/salt/orchestrate/runit.sls``.
+
+Types of Reactions
+==================
 
-Writing SLS Files
------------------
+==============================  ==================================================================================
+Name                            Description
+==============================  ==================================================================================
+:ref:`local <reactor-local>`    Runs a :ref:`remote-execution function <all-salt.modules>` on targeted minions
+:ref:`runner <reactor-runner>`  Executes a :ref:`runner function <all-salt.runners>`
+:ref:`wheel <reactor-wheel>`    Executes a :ref:`wheel function <all-salt.wheel>` on the master
+:ref:`caller <reactor-caller>`  Runs a :ref:`remote-execution function <all-salt.modules>` on a masterless minion
+==============================  ==================================================================================
+
+.. note::
+    The ``local`` and ``caller`` reaction types will be renamed for the Oxygen
```

Comment:
This sort of information should be moved to or mirrored in the release notes for Oxygen.

Reply:
We have no .rst file for Oxygen in the 2017.7 branch. We don't typically create one until we've branched, and then typically in develop rather than the release branch.

I can take care of this once this PR is merged and merged forward into develop, as I'd like to link to the "Reaction Types" section from the release notes and it won't exist until we merge this PR forward.